ecs-init/docker/docker_unspecified.go

```diff
@@ -40,7 +40,16 @@ func createHostConfig(binds []string) *godocker.HostConfig {
 		dhclientLibDir+":"+dhclientLibDir+readOnly,
 		dhclientExecutableDir+":"+dhclientExecutableDir+readOnly)
 
+	// Bit ugly adding the log rotation configuration here inline, consider
+	// revising to make this easier to read.
 	return &godocker.HostConfig{
+		LogConfig:   godocker.LogConfig{
+			Type: "json-file",
+			Config: map[string]string{
+				"max-size": "256m",
```

Comment:
This number seems more than reasonable to me; what is the reasoning behind this choice?

This arrangement (`8*256m`) will still result in some slightly large log files totalling 2GB, and AFAIK this is duplicated log output that the agent writes to `/var/log/ecs`. There may be some tools out there using `docker logs` to obtain these logs so they definitely could be taking advantage of the slack - however if they are consuming near-realtime then retaining 2GB of duplicated logs may be unnecessary. 

It may be prudent to start with smaller values and allow the knobs to be tuned here for use cases where a larger retention policy is needed. What do you think about that approach?

Reply:
Seems reasonable. Have changed the starting values to 16m, 4 files and put them in a configuration item in ecs-init/config/common.go